.bumpversion.cfg

```diff
@@ -1,7 +1,6 @@
 [bumpversion]
-current_version = 0.17.1
+current_version = 0.17.2
 commit = True
 tag = True
 
 [bumpversion:file:cl_sii/__init__.py]
-
```

Comment:
This was probably removed by the [EditorConfig](https://editorconfig.org/), did you use [bumpversion](https://pypi.org/project/bumpversion/) to update the version?

Reply:
Yes, i ran the command `bumpversion --new-version '0.17.2' minor`